### PR TITLE
Export answers to task questionnaires

### LINKED ISSUE #39: Allow export answers for a questionnaire
- [ ] make this work:
image

### PR DESCRIPTION
**Pull request by verarojman**
Closes #39 

- [x] Discuss whether questionnaire should be attached to the component instead of tasks﻿ (@microstudi)

~~**If questionnaire must be attached to task:**~~
~~- [ ] Find way to filter the answers being exported (currently they all get mixed up)~~

**Once that is done**
- [x] Add specs

### CODE REVIEW

**Comment by microstudi** [comment]
If we attach it to the component, all the tasks will have to share the same questionnaire, is this a good think? it is not?
I find useful to be able create different forms but if it raises too much problems (like complicated exports), let's do it and simplify.

File: app/models/decidim/time_tracker/activity.rb [hunk 1]
```diff
@@ -100,7 +101,7 @@ def answered_by?(user)
 
       # used as a unique idenfier when answering the task associated questionnaire
       def session_token(user)
-        "#{user.id}-#{id}"
+        "#{Digest::SHA1.hexdigest(user.id.to_s)}-#{id}"
```
Comment (microstudi) [inline app/models/decidim/time_tracker/activity.rb:104]: I don't know about this, is pretty easy to disclose the user.id anyway, you just need to loop integers to do it.
I we want to do this, we should use this: https://github.com/decidim/decidim/pull/6850

Not available in 0.22 yet though

Comment (microstudi) [reply app/models/decidim/time_tracker/activity.rb:104]: actually, not available until release 0.24 probably

Comment (microstudi) [reply app/models/decidim/time_tracker/activity.rb:104]: For the moment, I think we can simply not export or show this information anywhere public

File: app/models/decidim/time_tracker/assignee_questionnaire.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Decidim
+  module TimeTracker
+    class AssigneeQuestionnaire < ApplicationRecord
```
Comment (microstudi) [inline app/models/decidim/time_tracker/assignee_questionnaire.rb:5]: is this for creating questionnaires for gender/age for the assignee?
I'm not sure if this information should simply be 2 fields in the assignee table. Let's discuss this

Comment (verarojman) [reply app/models/decidim/time_tracker/assignee_questionnaire.rb:5]: Yes, that's probably the most wise approach right now. We can always add something like this later.

**Review by microstudi — APPROVED** [review]
SUPER @verarojman !!
Many thanks, this goes in!

**Pull request merged**